### ISSUE TITLE
Site does not deploy to Netlify?

### ISSUE THREAD

**Issue by andaydvice** [stackblitz/bolt.new#4079]
### Describe the bug

Cannot deploy after 5 attempts

Command failed
npm run dev

Command failed
npm run build

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-cnrnnby6

### Steps to reproduce

Press the deploy button

### Expected behavior

Deploy to Netlify seamlessly

### Screen Recording / Screenshot

https://app.screencast.com/MD3jXixhUZ4kk

### Platform

```
Browser name = Chrome
Full version = 131.0.0.0
Major version = 131
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 82418471,
  "usedJSHeapSize": 78698787,
  "jsHeapSizeLimit": 4294705152
}
Username = andaydvice
Chat ID = 64a30902a835
```

**Comment by andaydvice** [comment]
Bolt tried to fix problems 5 times but failed. To prevent spending more tokens, you might want to try to fix them manually or rollback to a previous checkpoint instead.

**Comment by andaydvice** [comment]
Yes I did roll it back and another 5 attempts failed wasting yet more credits because of errors

**Comment by diegoelamin** [comment]
This happened to me as well. Also it seems like Bolt will keep trying the same fix over and over again. Also, Also, there seems to a lot of type errors.

Here my current workflow, Copy and paste the error into Claude or ChatGPT, Copy and paste the AI suggested fix back into Bolt. Claude and ChatGPT always seem to find different fixes than Bolt.
